src/figuresClasses.ts

```diff
@@ -1,19 +1,81 @@
+enum Shapes {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle'
```

Comment:
```suggestion
  Triangle = 'triangle';
  Circle = 'circle';
  Rectangle = 'rectangle';
```

Reply:
gives an error
An enum member name must be followed by a ',', '=', or '}'.ts(1357)

Reply:
You are right. Keep the correct option.